qualtran/_infra/data_types.py

```diff
@@ -528,6 +592,13 @@ def from_bits(self, bits: Sequence[int]) -> Fxp:
         fxp_bin = "0b" + bits_bin[: -self.num_frac] + "." + bits_bin[-self.num_frac :]
         return Fxp(fxp_bin, dtype=self.fxp_dtype_str)
 
+    def from_bits_array(self, bits_array: NDArray[np.uint8]):
+        assert isinstance(self.bitsize, int), "cannot convert to bits for symbolic bitsize"
+        # TODO figure out why `np.vectorize` is not working here
```

Comment:
Add a link to an open issue with the TODO?

Reply:
I'm planning to remove the `fxpmath` dependency in classical sim for now, in a follow-up PR, so this might become irrelevant. I can perhaps open an issue later when we decide to actually use fxpmath as our floating point library.

Reply:
You want to remove all code that uses `fxpmath` and remove the `QFxp` type ?

Reply:
No no, the `QFxp` type will stay, but will use integer values (i.e. binary repr of the fxp) for the classical simulator, instead of `Fxp`.

And other uses of `fxpmath` in the code will not be touched, e.g. the current phase gradient `scaled_val` etc.

Reply:
So whenc calling `bloq.call_classically(phase_grad=_fxp_value_)`; the `_fxp_value_` would be an int I believe ?

Reply:
Yep, like it is right now. I'll add a helper method to `QFxp` to convert a float/Fxp value into the correct int value for passing to the classical sim.